package-build.yml

```diff
@@ -15,8 +15,7 @@ spec:
           template:
           - ytt:
               paths:
-              - config
-              - config-release
+              - config/package-bundle/config
```

Comment:
Are we able to keep this as `config`?

Why? There are third party consumers of our OSS packages that have other overlays and things in the config folder, and changing this structure would affect them. Ideally we don't need to subdir down to this package-bundle folder.

Reply:
That makes sense. I tried to keep the structure same as secretgen-controller, but it would make more sense to keep it compatible with the existing structure for kc package.

Reply:
Done